src/components/Form/Toggle.tsx

```diff
@@ -93,7 +93,6 @@ const componentOrElementWithProps = (
       props,
     ); // component
   }
-  return;
```

Comment:
Is this and other similar changes here because of the linter change? What changed in the rules?

Reply:
The rules are now aligned with the design system repo and are more strict in some areas. So yes it is directly related to changes in ruleset. There are additional changes required which are now suppressed as warnings in the linter config.